2021-08-13 14:22:16,664 INFO:  * Running on http://0.0.0.0:4001/ (Press CTRL+C to quit)
2021-08-13 14:22:17,042 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:17,044 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:17] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:17,043 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/s2021-08-13 14:22:17,148 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:17] "[37mPOST /getChatRoomList HTTP/1.1[0m" 200 -
2021-08-13 14:22:18,157 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:18] "[37mPOST /getChatRoomList HTTP/1.1[0m" 200 -
2021-08-13 14:22:19,151 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:19] "[37mPOST /getChatRoomList HTTP/1.1[0m" 200 -
   raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:17,045 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:17] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:18,050 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:18,051 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:18] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:18,050 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:18,053 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:18] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:19,039 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:19,041 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:19] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:19,039 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:19,043 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:19] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:19,357 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:19,358 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:19] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:20,045 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:20,046 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:20] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:20,160 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:20,161 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:20] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:21,042 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:21,044 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:21] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
2021-08-13 14:22:21,046 ERROR: Exception on /notify [POST]
Traceback (most recent call last):
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/C217_yochi/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app4001.py", line 97, in notify
    notifyRes = Controller.notification(RoomIDList)
AttributeError: module 'Controller4000' has no attribute 'notification'
2021-08-13 14:22:21,047 INFO: 23.98.71.39 - - [13/Aug/2021 14:22:21] "[35m[1mPOST /notify HTTP/1.1[0m" 500 -
